client/src/locales/fr-FR.json

```diff
@@ -52,6 +52,10 @@
             "joinByLink": "Rejoindre par lien",
             "joinByQRcode": "Rejoindre par QR code"
         },
+        "loginNotification": {
+            "title": "Connexion échouée !",
```

Comment:
I would change that by `Impossible de se connecter`. It feels less like a straight-up translation from English.